<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/XD-OCR.png</normaloff>:/resource/XD-OCR.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lb_show">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;显示图像&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>识别结果：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="tb_text">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(33, 150, 243);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_captured">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;拍摄的照片将显示在这里&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_openCamera">
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 8px; font-weight: bold; background-color: #ffff8f; color: black; }</string>
        </property>
        <property name="text">
         <string>打开摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_captureImage">
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 8px; font-weight: bold; background-color: #7bffb6; color: black; }</string>
        </property>
        <property name="text">
         <string>拍照识别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_selectImage">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 8px; font-weight: bold; background-color: #61c2ff; color: black; }
</string>
        </property>
        <property name="text">
         <string>选择图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_saveText">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 8px; font-weight: bold; background-color: #a3ff5d; color: black; }
</string>
        </property>
        <property name="text">
         <string>保存文本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_toggleSaveMode">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 8px; font-weight: bold; background-color: #ff7dd8; color: black; }
</string>
        </property>
        <property name="text">
         <string>保存模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_singleDoc">
        <property name="text">
         <string>单文档模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_multiDoc">
        <property name="text">
         <string>多文档模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_showRect">
        <property name="text">
         <string>显示文本检测框</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
